### PR TITLE
Boolean input tag (issue fixes)

### LINKED ISSUE #1575: Boolean input should indicate it has the focus
The new boolean input should give an indication that it has the focus.  Maybe if we made it smaller, it could have an outline when it is focused, as other elements have.

### LINKED ISSUE #1576: Boolean input display problem with full checkwork button
If adding a full checkwork button to a boolean input, the checkwork button overlaps with the checkbox.

`<answer type="boolean" forceFullCheckworkButton>true</answer>`

### PR DESCRIPTION
**Pull request by keaganeng**
Fixes #1575 
Fixes #1576

### CODE REVIEW

**Comment by kevincharles** [comment]
Please update line 92 to go back to ID's not name
  `const inputKey = name + '_input';`

File: src/Viewer/renderers/booleanInput.jsx [hunk 1]
```diff
@@ -229,10 +229,11 @@ export default React.memo(function BooleanInput(props) {
         disabled={disabled}
       />
       <span className="checkmark"></span>
-      <span style={{marginLeft: "8px"}}>{label}</span>
+      {label != "" ? <span style={{marginLeft: "2px"}}>{label}</span> : <span>{label}</span>}
     </label>
     {checkWorkButton}
   }
+  console.log(label);
```
Comment (dqnykamp) [inline src/Viewer/renderers/booleanInput.jsx:236]: Delete or comment out this line

File: src/Viewer/renderers/booleanInput.jsx [hunk 2]
```diff
@@ -89,7 +89,7 @@ export default React.memo(function BooleanInput(props) {
 
   let disabled = SVs.disabled;
 
-  const inputKey = id + '_input';
+  const inputKey = name + '_input';
```
Comment (dqnykamp) [inline src/Viewer/renderers/booleanInput.jsx:92]: Revert this change.

**Review by dqnykamp — CHANGES_REQUESTED** [review]

File: src/Viewer/renderers/booleanInput.jsx (hunk 2 above)
Comment (keaganeng) [reply src/Viewer/renderers/booleanInput.jsx:92]: Done

File: src/Viewer/renderers/booleanInput.jsx (hunk 1 above)
Comment (keaganeng) [reply src/Viewer/renderers/booleanInput.jsx:236]: Done

**Pull request merged**